scripts/README.md

```diff
@@ -20,6 +23,23 @@ A node script to generate markdown files required for hosting Learning Path on i
 ```
 npm ci
 node generate_learning_path_markdown.js
+```
+
+## How to update innersourcecommons.org with new articles
+
+Anytime there are _new_ articles written, you need to run the script that generates the new pages on the [innersourcecommons.org] site.
```

Comment:
There's an edge case if an existing article's title gets updated (e.g. https://github.com/InnerSourceCommons/innersourcecommons.org/pull/214) but perhaps not worth calling out.

Reply:
Yeah I thought about that, too.